flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/partition/ClusterPartitionReport.java

```diff
@@ -57,12 +59,21 @@ public static class ClusterPartitionReportEntry implements Serializable {
 
         private final IntermediateDataSetID dataSetId;
         private final Set<ResultPartitionID> hostedPartitions;
+        private final Set<ShuffleDescriptor> shuffleDescriptors;
         private final int numTotalPartitions;
 
         public ClusterPartitionReportEntry(
```

Comment:
nit: Can we remove this? If it is only used by tests, maybe add a VisibleForTesting annotation.